OSDOCS-20733: CQA fixes for ROSA HCP Tutorials 1-6 (#115626)
openshift/openshift-docs: modules/aws-account-billing-selection.adoc

.Procedure

> * When using the web interface for deploying {product-title}, verify that the *AWS billing account* drop-down is set to the AWS account that accepted the private offer, providing the purchased quota is intended to be used by the cluster that is being created.

Reviewer: Clarity question: 

Do users verify the  AWS billing account is set to the private offer SO THAT the purchased quota  is used by the cluster that is being created? 

If so, maybe something like this: 

```suggestion
* When using the web interface for deploying {product-title}, verify that the *AWS billing account* drop-down is set to the AWS account that accepted the private offer, so that the cluster that is being created uses the purchased quota.
```

Reviewer: Or, maybes something like this: 

```suggestion
* When using the web interface for deploying {product-title}, verify that the *AWS billing account* drop-down is set to the AWS account that accepted the private offer, ensuring that the purchased quota is getting used by the cluster that is being created.
```

Reviewer: Basically, I read "providing the purchased quota" as a second action you wanted the user to do, but I don't think that it is.
